Packs/FiltersAndTransformers/Scripts/IfElif/IfElif.py

```diff
@@ -58,12 +99,26 @@ def load_conditions(self, conditions):
             r" from_context('\1')",
             conditions
         )
-        self.conditions = self.evaluate(conditions)
+        try:
+            self.conditions = self.evaluate(conditions)
+        except SyntaxError as e:
+            raise SyntaxError(
+                f'Cannot load JSON. Invalid syntax at line: {e.args[1][1]}; position: {e.args[1][2]}'
+            ) from e
+
+    def validate_conditions(self):
+        for i, d in enumerate(self.conditions, 1):
+            if 'condition' not in d:
+                raise ValueError(f'Condition {i} has no key "condition".')
+            elif 'return' not in d:
+                raise ValueError(f'Condition {i} has no key "return".')
+            elif not isinstance(d, dict):
+                raise ValueError(f'Condition {i} is not a dictionary.')
 
     def get_value(self, node):
         match type(node):
             case ast.Name:
-                return json.loads(node.id)
+                return self.variables[node.id]
```

Comment:
ast.Name indicate a variable so it makes sense to have a dict with the supported variables (right now it holds json values but it could have anything in the future)